src/common/lib/client/auth.ts

```diff
@@ -116,7 +115,7 @@ class Auth {
   client: BaseClient;
   tokenParams: API.TokenParams;
   currentTokenRequestId: number | null;
-  waitingForTokenRequest: ReturnType<typeof Multicaster.create> | null;
+  waitingForTokenRequest: ReturnType<typeof Multicaster.create<API.TokenDetails>> | null;
```

Comment:
any reason not to do this?
```suggestion
  waitingForTokenRequest: MulticasterInterface<API.TokenDetails> | null;
```

Reply:
Nope, guess changing it just didn't seem in scope of the PR, but have added 9eeb7a2e1f32e2e186b317ab5d709dc6cf5666f7.